src/ecdsa/ellipticcurve.py

```diff
@@ -86,6 +86,9 @@ def __eq__(self, other):
     else:
       return False
 
+  def __neg__(self):
+    return Point(self.__curve, self.__x, self.__curve.p() - self.__y)
+
```

Comment:
why point negation is needed?

Reply:
Not really for the fix, but the scalarmult can handle negative scalars now.